quirks/websites-with-shared-credential-backends.json

```diff
@@ -248,5 +248,9 @@
         "cooksillustrated.com",
         "cookscountry.com",
         "onlinecookingschool.com"
+    ],
+    [
+        "udel.edu",
+        "microsoftonline.com"
```

Comment:
Can you explain this? Are there login pages that both udel.edu and microsoftonline.com that accept the same credentials? This seems a little off to me.

Reply:
The second URL should be https://www.office.com/. This is because Microsoft Office 365 Accounts with the same credentials as UDel Net IDs are automatically provisioned in the back-end.